docs/products/kafka/howto/provectus-kafka-ui.rst

```diff
@@ -0,0 +1,88 @@
+Use Provectus® UI for Apache Kafka® with Aiven for Apache Kafka®
+================================================================
+
+`Provectus® UI for Apache Kafka® <https://github.com/provectus/kafka-ui>`_ is a popular Open-Source web GUI for Apache Kafka® management that allows you to monitor and manage Apache Kafka® clusters.
+
+Prerequisites
+-------------
+
+To connect Provectus® UI for Apache Kafka® to Aiven for Apache Kafka® you need to create a :doc:`Java keystore and truststore containing the service SSL certificates <keystore-truststore>`. 
+
+Furthermore, you need to collect the following information:
+
+* ``APACHE_KAFKA_HOST``: The Aiven for Apache Kafka hostname
+* ``APACHE_KAFKA_PORT``: The Aiven for Apache Kafka port
+* ``KEYSTORE_FILE_NAME``: The name of the Java keystore containing the Aiven for Apache Kafka SSL certificates
+* ``TRUSTSTORE_FILE_NAME``: The name of the Java truststore containing the Aiven for Apache Kafka SSL certificates
+* ``SSL_KEYSTORE_PASSWORD``: The password used to secure the Kava keystore
+* ``SSL_KEY_PASSWORD``: The password used to secure the Kava key
+* ``SSL_TRUSTSTORE_LOCATION``: The password used to secure the Kava truststore
+* ``SSL_TRUSTSTORE_PASSWORD``: The password used to secure the Kava truststore
+* ``SSL_STORE_FOLDER``: The absolute path of the folder containing both the truststore and keystore
+
+
+Run Provectus® UI for Apache Kafka® on Docker or Podman
+-------------------------------------------------------
+
+
+Share keystores with non-root user
+''''''''''''''''''''''''''''''''''
+
+Since container for Provectus® UI for Apache Kafka® uses non-root user, to avoid permission problems, while keeping the secrets safe, perform the following steps (see example commands below):
+
+1. Create separate directory for secrets::
+
+    mkdir kafka-secrets
```

Comment:
I just thought that maybe it's worth noting that the name of the directory can be arbitrary.